### PR TITLE
Fix: correctly save/load key bindings and wire subcommands

### PR DESCRIPTION
**Pull request by Alzakath**
Summary

    Fixes a bug where the "bindings" subcommands only logged an action instead of actually calling BindingsManager:Save() and BindingsManager:Load().
    Improves safety and robustness when saving/loading key bindings.

What changed

    BindingsManager.Save now:
        Skips empty / nil keys when saving.
        Normalizes empty-string keys to nil.
        Saves the current binding set id (RuilhennDB.bindingSet) so Load can restore the same set.
        Ensures RuilhennDB exists before writing.
        Uses a user-visible Info log for success messages (falls back to a literal string if localization is missing).
    BindingsManager.Load now:
        Checks for the presence of saved bindings before attempting to load.
        Only calls SetBinding when a saved key exists.
        Calls SaveBindings with the saved binding set id (or the current set if none was saved).
        Uses a user-visible Info log for success messages.
    Subcommand wiring:
        The "load" and "save" subcommand handlers now call BindingsManager:Load() and BindingsManager:Save(), respectively (previously they only logged).

Files changed

    BindingsManager.lua — replaced Save/Load implementations and fixed subcommand wiring.

Testing notes / QA checklist

    Run the save command and verify RuilhennDB.keyBindings is created and populated.
    Verify RuilhennDB.bindingSet is set to the current binding set id when saved.
    Run the load command and confirm bindings are reapplied (check that SetBinding is invoked only for non-empty saved keys).
    Confirm SaveBindings is called with the saved binding set id (or with the current set if no id was saved).
    Test edge cases:
        No saved bindings exist (Load should exit cleanly and log a debug message).
        Saved entries contain empty strings (these should be ignored / normalized).
        Conflicting existing bindings (verify current behavior or note in review if an overwrite/clear strategy is desired).
    Verify that running the commands via the UI/console that route through Commands["bindings"] with subcommands "save" and "load" actually execute the behavior (not just log).

Backward compatibility & risks

    This change only modifies how bindings are saved/loaded and how the subcommands are wired; it should be backward-compatible.
    Risk: loading saved bindings can overwrite existing user bindings. Consider whether an explicit confirmation/overwrite strategy is needed for end users.

Suggested reviewers / labels

    Reviewers: someone familiar with keybinding behavior and the addon command/persistence system.
    Labels: bug, enhancement, needs-testing

Notes / follow-ups

    Consider an option to clear or unbind conflicting keys before applying saved bindings (prevents duplicate binding collisions).
    Consider storing whether the bindings were character-specific vs account-wide if your addon needs to support both modes.

### CODE REVIEW

File: BindingsManager.lua [hunk 1]
```diff
@@ -10,46 +10,80 @@ local SetBinding = SetBinding
...
+            })
+        end
     end
+
     RuilhennDB.keyBindings = bindings
-    ns.Log:Debug(L["KEY_BINDINGS_SAVED"])
+    RuilhennDB.bindingSet = bindingSet
+
+    -- Use Info for user-visible messages; fall back to a plain string if localization is missing
+    ns.Log:Info(L["KEY_BINDINGS_SAVED"] or "Key bindings saved.")
```
[the remark this answers is not in the document]
Comment (Alzakath) [reply BindingsManager.lua]: fixed in latest commit
